src/FluffySpoon.AspNet.LetsEncrypt/LetsEncryptRenewalService.cs

```diff
@@ -141,7 +141,15 @@ private async Task AcquireNewCertificateForDomains(string[] domains)
 		{
 			_logger.LogInformation("Ordering LetsEncrypt certificate for domains {0}.", new object[] { domains });
 
-			var order = await acme.NewOrder(domains);
+		        IOrderContext order = null;
```

Comment:
Why allow order to be `null` here?

Instead, extend the try/catch so it reaches further.